### PR TITLE
Switch Katello bats tests to better repo (fix Katello nightly BATS failures)

### PR DESCRIPTION
**Pull request by ianballou**
`rpm-packages-updateinfo` contains an important erratum (the SEA one)  that points to the wrong RPMs. `rpm-no-comps` correctly points to walrus-5.21.

Also added some repo cleanup since the Pulp fixtures reuse RPMs, which causes clashes between the modular and non-modular versions of the RPMs.

Should fixes the Katello nightly pipeline.

Tested against nightly.

### CODE REVIEW

**Comment by ehelms** [comment]
@ianballou looks good, can you address the shell check comments?

**Comment by ianballou** [comment]
Doing a final full test and then I'll push up the last changes

**Comment by ianballou** [comment]
@ehelms this is good to go now, finished up my testing.

File: bats/export_repos [hunk 1]
```diff
@@ -0,0 +1 @@
+Zoo
```
Comment (ehelms) [inline bats/export_repos:1]: Is this file needed?

Comment (ianballou) [reply bats/export_repos:1]: Might've snuck in, lemme double check.

Comment (ianballou) [reply bats/export_repos]: alright, it's deleted.

**Review by ehelms — APPROVED** [review]

**Comment by evgeni** [comment]
For some reason EL9 is failing with this

https://jenkins-foreman.apps.ocp.cloud.ci.centos.org/blue/organizations/jenkins/foreman-pipeline-katello-rpm-nightly/detail/foreman-pipeline-katello-rpm-nightly/332/pipeline

**Comment by ehelms** [comment]
Yea - see https://github.com/theforeman/foreman-packaging/pull/10669

**Comment by evgeni** [comment]
Oooh, that's related?!

**Comment by ehelms** [comment]
> Oooh, that's related?!

Yep, check the output:

```

[2024-04-04T23:50:20.340Z]     # $ hammer repository delete --organization="${ORGANIZATION}" --product="${PRODUCT}" --name="${YUM_REPOSITORY_2}" --remove-from-content-view-versions=true

[2024-04-04T23:50:20.340Z]     # Repository deleted.

[2024-04-04T23:50:20.340Z]     # $ hammer repository delete --organization="${ORGANIZATION}" --product="${PRODUCT}" --name="${YUM_REPOSITORY_3}" --remove-from-content-view-versions=true

[2024-04-04T23:50:20.340Z]     # Repository deleted.

[2024-04-04T23:50:20.340Z]     # $ echo "::ForemanTasks.sync_task(::Actions::Katello::OrphanCleanup::RemoveOrphans, SmartProxy.pulp_primary)"

[2024-04-04T23:50:20.340Z]     # $ foreman-rake console

[2024-04-04T23:50:20.340Z]     # rake aborted!

[2024-04-04T23:50:20.340Z]     # LoadError: cannot load such file -- irb
```

**Comment by evgeni** [comment]
What a way to find a bug ;)
